charts/dgu/templates/redis/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Wouldn't redis need to be a statefulset? (You'll get away with it if you only have one replica, but it'd still make sense to use statefulset anyway so that e.g. you could use this in prod later without the hassle of changing the nonprod ones.)

Reply:
Also there's probably a chart you could reuse for Redis — definitely worth a look.

Reply:
Heh, I was about to suggest taking a look at how the Argo CD chart does it, but turns out they just roll their own and they even use Deployment, so I guess it's fine? 😄

Reply:
I guess if you don't need persistence or replication (maybe you don't?) then Deployment is just fine. Sorry didn't mean to divert your attention to this.